decentraland/unity-renderer
# fix: avatar in ghost mode if an owned emote is not found

## What does this PR change?

Fixes #4584 

If you are using the builder and

1. Add an emote using `WITH_ITEMS` or `WITH_COLLECTIONS` 
2. Equip in the wheel and run said emote
3. Refresh the website but without the `WITH_ITEMS` or `WITH_COLLECTIONS` 

You will get a ghost avatar since the emote promise fail since you dont own anymore an emote thats already equipped. This PR adds a catch to avoid the failure of the avatar curation


## How to test the changes?

1. Go to https://play.decentraland.org/?explorer-branch=fix/ghost-mode-after-removing-emote-collection&BUILDER_SERVER_URL=https%3A%2F%2Fbuilder-api.decentraland.org%2Fv1&NETWORK=goerli&DEBUG_MODE=true&position=100,100&realm=loki&WITH_COLLECTIONS=e0081577-dfb2-4fc3-bc0f-5f70be79733d
2. Equip the YOGA custom emote. Play it
3. Go to https://play.decentraland.org/?explorer-branch=fix/ghost-mode-after-removing-emote-collection&BUILDER_SERVER_URL=https%3A%2F%2Fbuilder-api.decentraland.org%2Fv1&NETWORK=goerli&DEBUG_MODE=true&position=100,100&realm=loki
(`WITH_COLLECTIONS` is gone)
4. You avatar should load normally

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
